.github/workflows/docker-publish.yaml

```diff
@@ -0,0 +1,60 @@
+name: Docker
+
+on:
+  push:
+    tags:
+      - v*
```

Comment:
@bluemellophone This is almost the same as the nightly workflow, isn't it possible to merge this into nightly.yml?

Reply:
Yes, I left it a separate file so that we could manage them separately.  I don't mind merging them